src/main/java/interview/jana/crawler/FindEmailIds.java

```diff
@@ -0,0 +1,84 @@
+package interview.jana.crawler;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import interview.jana.caches.EmailStoreCache;
+import interview.jana.caches.InMemoryCache;
+import interview.jana.caches.InMemoryEmailStore;
+import interview.jana.caches.PageCache;
+import interview.jana.caches.RedisCache;
+import interview.jana.processors.JedisTaskProcessor;
+import interview.jana.processors.RunnableProcessor;
+import interview.jana.processors.TaskProcessor;
+
+/**
+ * Main Class used to configure the caches and Executors
+ * 
+ * @author Anusha
+ *
+ */
+// TODO Need to use proper Factories for initializing
+public class FindEmailIds {
+
+	public static ExecutorService executor = Executors.newSingleThreadExecutor();
+	public static PageCache cache;
+	public static String homeURL;
+	public static EmailStoreCache emailStore;
+
+	public static void main(String args[]) {
+
+		initialize(args[0]);
+		String url = "http://" + homeURL;
+		System.out.println("Found Processing:" + url);
+		UseJedisPubsub(url);
```

Comment:
I was thinking of something along the lines of just shutting down app in the case where Redis is not available, giving some kind of message to alert the user as to why it's not running.